src/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# Uncomment for VERBOSE Makefiles. Useful for debugging.
+#set(CMAKE_VERBOSE_MAKEFILE ON)
+
+# project name and language
+project(ImpactT LANGUAGES Fortran)
+
+option(USE_MPI "Activate MPI build" OFF)
+
+
+list(APPEND _sources
+    DataStruct/NumConst.f90
+    DataStruct/PhysConst.f90
+    DataStruct/Pgrid.f90
+	DataStruct/Data.f90
```

Comment:
You seem to be mixing tabs and spaces in this list :-)

Reply:
That was a stupid mistake on my configuration... I will get that fixed... thank you for catching it.

Reply:
Addressed via https://github.com/impact-lbl/IMPACT-T/pull/15/commits/4e2629925788489a5862b721a0e9d968a7472e49